packages/tpetra/core/src/Tpetra_CrsGraph_decl.hpp

```diff
@@ -1838,11 +1838,40 @@ namespace Tpetra {
     size_t
     insertGlobalIndicesImpl (const RowInfo& rowInfo,
                              const GlobalOrdinal inputGblColInds[],
-                             const size_t numInputInds);
+                             const size_t numInputInds,
+                             std::function<void(const size_t, const size_t, const size_t)> fun =
+                                 std::function<void(const size_t, const size_t, const size_t)>());
 
     void
```

Comment:
Hm, how come this one returns `void` instead of `size_t`?  (Don't have to change for now.)

Reply:
Which, the std::function callback?  If so, why would it return anything but void?  The caller never uses the return value

Reply:
@tjfulle I mean the outer method, `insertLocalIndicesImpl`.  `insertGlobalIndicesImpl` returns `size_t`, but `insertLocalIndicesImpl` returns `void`.

Reply:
@mhoemmen Ah, I see.  I don’t know why, I’d have to redirect that question back to you!

Reply:
@tjfulle I don't know why either ;-)  We should fix that at some point.